### PR TITLE
integration tests on windows

### PR DESCRIPTION
**Pull request by solarw**
check integration tests works well on windows platform

### CODE REVIEW

**Comment by 5A11** [comment]
We used to exclude windows from integration tests because of the docker client we were running for integration tests and there were some problems with docker on windows. Now that we are not running integration tests against a local node docker image we can reenable integration tests for windows.

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -76,3 +72,23 @@ jobs:
...
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-python@master
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: pip install tox==3.24.1
+    - name: Integration tests
+      run: |
+          tox -e test-integration
```
[the remark this answers is not in the document]
Comment (solarw) [reply .github/workflows/workflow.yml:94]: it will be implemented after https://github.com/fetchai/cosmpy/pull/227 merged

**Review by jrriehl — APPROVED** [review]
LGTM

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**